gencertbuf.pl

```diff
@@ -44,6 +45,14 @@
         [ "./certs/ed25519/client-ed25519-key.der", "client_ed25519_key" ]
         );
 
+# x25519 keys and certs
+# Used with HAVE_CURVE25519 define.
```

Comment:
Let's make a new define for `test.c` called `USE_CERT_BUFFERS_25519` so that people who don't need/want them don't automatically get these pulled in along with `HAVE_CURVE25519` only.